Objective-C/Tests/MessageEndpointTest.m

```diff
@@ -171,7 +171,7 @@ - (void)tearDown {
 }
 
 - (void) startDiscovery {
-    _serverConnected = [self expectationWithDescription: @"Server Connected"];
+    _serverConnected = [self allowOverfillExpectationWithDescription: @"Server Connected"];
```

Comment:
Is it a bug that is causing multiple filling?

Reply:
This might be, I will create a ticket and add a todo for now, will work in parallel. 
https://issues.couchbase.com/browse/CBL-3699

Reply:
I believe, it started happening with the Xcode upgrade. I am trying to reproduce the issue locally(command line), it won't happen when running through Xcode.